build.gradle.kts

```diff
@@ -84,20 +89,17 @@ val gitStatus = try {
 }
 
 gradlePlugin {
+    website.set(productUrl)
+    vcsUrl.set("${productUrl}.git")
```

Comment:
Based on [the example values in the docs](https://docs.gradle.org/current/userguide/publishing_gradle_plugins.html#configure_the_plugin_publishing_plugin):
> ```
> website = "https://github.com/ysb33r/gradleTest"
> vcsUrl = "https://github.com/ysb33r/gradleTest.git"
> ```